cms/static/js/views/__feedback.js

```diff
@@ -1,4 +1,4 @@
-define(["jquery", "underscore", "underscore.string", "backbone", "js/utils/templates"],
+define(["jquery", "underscore", "underscore.string", "backbone", "common/js/components/utils/templates"],
```

Comment:
I don't think you intended to merge these files since you aren't using them. I would remove these three files and then move the two feedback files directly to common so that the history is preserved (don't bring templates over).